README.md

```diff
@@ -4,8 +4,14 @@
 
 WARNING: if you fork this repo, github actions will run daily on it. To disable this, go to <myuser>/examples/settings/actions and Disable Actions for this repository.
 
-A repository showcasing examples of using [PyTorch](https://github.com/pytorch/pytorch):
+`pytorch/examples` is a repository showcasing examples of using [PyTorch](https://github.com/pytorch/pytorch). The goal is to have curated, short, *high quality* examples that are substantially different from each other that can be plugged into your existing work.
```

Comment:
> can be plugged into your existing work
Is that what "recipes" ought to do? Should the examples instead "highlight best practices that can be emulated in your existing work?"

Reply:
Yeah I like this phrasing